ops/model.py

```diff
@@ -453,8 +452,8 @@ def _calculate_expiry(expire: Optional[Union[datetime.datetime, datetime.timedel
 class Unit:
     """Represents a named unit in the model.
 
-    This might be your unit, another unit of your application, or a unit of another application
-    that you are related to.
+    This might be this charm's unit, another unit of the charm's application, or a unit of
```

Comment:
Another instance where "current unit" applies:

```suggestion
    This might be the current unit, another unit of the charm's application, or a unit of
```